rs/bitcoin/ckbtc/minter/src/updates/tests.rs

```diff
@@ -52,6 +44,18 @@ mod update_balance {
         );
         mock_schedule_now_process_logic(&mut runtime);
 
+        mutate_state(|s| {
+            audit::ignore_utxo(
+                s,
+                ignored_utxo.clone(),
+                account,
+                NOW.checked_sub(DAY).unwrap(),
+                &runtime,
+            )
+        });
+        mutate_state(|s| s.check_fee = ignored_utxo.value - 1);
+        let events_before: Vec<_> = storage::events().map(|event| event.payload).collect();
```

Comment:
nit: Here and below, for readability I would somewhat preserve the previous ordering, meaning I would do something like
```rust
        init_state_with_ecdsa_public_key();
        let account = ledger_account();
        let mut runtime = MockCanisterRuntime::new();
        mock_increasing_time(&mut runtime, NOW, Duration::from_secs(1));

        let ignored_utxo = ignored_utxo();
        mutate_state(|s| {
            audit::ignore_utxo(
                s,
                ignored_utxo.clone(),
                account,
                NOW.checked_sub(DAY).unwrap(),
                &runtime,
            )
        });
        mutate_state(|s| s.check_fee = ignored_utxo.value - 1);
        let events_before: Vec<_> = storage::events().map(|event| event.payload).collect();

       mock_get_utxos_for_account(&mut runtime, account, vec![ignored_utxo.clone()]);
        expect_check_transaction_returning(
            &mut runtime,
            ignored_utxo.clone(),
            CheckTransactionResponse::Failed(vec![]),
        );
        mock_schedule_now_process_logic(&mut runtime);
...
```
WDYT?